src/material.h

```diff
@@ -29,6 +30,9 @@ class material_type
         int _fire_resist;
         int _chip_resist;       // Resistance to physical damage of the item itself
         int _density;   // relative to "powder", which is 1
+        bool _edible;
+
+        mutable std::map<vitamin_id, double> _vitamins;
```

Comment:
> This is the correct and intended use for that modifier.

Got any source on that? It looks really ugly to allow modification of the map just to use the non-const operator on it, instead of just using a const-correct way of accessing the value.